crates/meroctl/src/cli/app/get.rs

```diff
@@ -0,0 +1,56 @@
+use clap::{Parser, ValueEnum};
+use eyre::{bail, Result as EyreResult};
+use reqwest::Client;
+
+use crate::cli::RootArgs;
+use crate::common::{get_response, multiaddr_to_url};
+use crate::config_file::ConfigFile;
+
+#[derive(Parser, Debug)]
+pub struct GetCommand {
+    #[arg(long, short)]
+    pub method: GetValues,
+
+    #[arg(long, short)]
+    pub app_id: String,
+}
+#[derive(ValueEnum, Debug, Clone)]
+pub enum GetValues {
+    Details,
+}
+
+impl GetCommand {
+    pub async fn run(self, args: RootArgs) -> EyreResult<()> {
```

Comment:
This is not specific to this PR, but I am curious as to why we use Eyre in some places and have more formalised errors in others. I think in this place it kinda makes sense as they are one-off errors. Just thinking out loud...

Reply:
> I am curious as to why we use Eyre in some places and have more formalised errors in others

ideally, we formalize errors in all libs and use eyre in bins. for now, eyre is an escape-hatch, as error-handling can get pretty involved and we don't care much for pattern-matching what happened, we mostly just report it.

> we formalize errors in all libs and use eyre in bins

this formalization will come eventually